[2024-03-11T14:00:10.174+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v3.third_task scheduled__2021-08-05T00:00:00+00:00 [queued]>
[2024-03-11T14:00:10.247+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v3.third_task scheduled__2021-08-05T00:00:00+00:00 [queued]>
[2024-03-11T14:00:10.251+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 6
[2024-03-11T14:00:10.912+0000] {taskinstance.py:2214} INFO - Executing <Task(BashOperator): third_task> on 2021-08-05 00:00:00+00:00
[2024-03-11T14:00:10.984+0000] {standard_task_runner.py:60} INFO - Started process 2821 to run task
[2024-03-11T14:00:11.126+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v3', 'third_task', 'scheduled__2021-08-05T00:00:00+00:00', '--job-id', '713', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpfbfejp5s']
[2024-03-11T14:00:11.181+0000] {standard_task_runner.py:88} INFO - Job 713: Subtask third_task
[2024-03-11T14:00:11.824+0000] {task_command.py:423} INFO - Running <TaskInstance: our_first_dag_v3.third_task scheduled__2021-08-05T00:00:00+00:00 [running]> on host 26e63b1d0d39
[2024-03-11T14:00:13.438+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='our_first_dag_v3' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2021-08-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-08-05T00:00:00+00:00'
[2024-03-11T14:00:13.441+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-11T14:00:13.444+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task3 and I ran after task1 and at the same time as task 2']
[2024-03-11T14:00:13.491+0000] {subprocess.py:86} INFO - Output:
[2024-03-11T14:00:13.509+0000] {subprocess.py:93} INFO - hey, I am task3 and I ran after task1 and at the same time as task 2
[2024-03-11T14:00:13.515+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-11T14:00:13.722+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v3, task_id=third_task, execution_date=20210805T000000, start_date=20240311T140010, end_date=20240311T140013
[2024-03-11T14:00:14.324+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-11T14:00:14.466+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
